NEWS

```diff
@@ -204,7 +205,15 @@ CHANGES WITH 243 in spe:
           send a DHCP RELEASE message when terminating.
 
         * systemd-networkd's DHCPv4 and DHCPv6 stacks can now be configured
-          seperately in the [DHCPv4] and [DHCPv6] sections.
+          separately in the [DHCPv4] and [DHCPv6] sections.
+
+        * systemd-networkd's DHCP support will now create an implicit host
+          route to the DNS server specified in the DHCP lease, in addition to
+          the routes listed explicitly in the lease. This should ensure that in
+          multi-homed systems DNS traffic leaves the systems on the interface
+          that acquired the DNS server information even if other routes such as
+          default routes exist. This behaviour may be turned off with the new
```

Comment:
`can be turned on`, as now it defaults to off, unfortunately.

Reply:
oh why? let's turn it on by default, it's so useful!

Reply:
It breaks Ubuntu CIs unfortunately.